jscomp/main/js_main.ml

```diff
@@ -533,6 +533,8 @@ let buckle_script_flags : (string * Bsc_args.spec * string) array =
     "-make-runtime-test", unit_call Js_packages_state.make_runtime_test,
     "*internal* make runtime test library";
 
+    "-bs-stop-after-cmj", set Js_config.cmj_only,
```

Comment:
tiny thing: maybe this flag could be called `-bs-stop-after-cmo` as it seems the idea is to move away from `cmj` and use `cmo` instead.

Reply:
Moving to cmo is just a hack not something we should ever merge. As that breaks a lot of assumptions.

Reply:
I see. I thought in a cross-compilation context, breaking this assumptions would be ok, as I understood build artifacts like `cmo` would be isolated.